packages/data-stores/src/auth/test/controls.ts

```diff
@@ -6,18 +6,22 @@
  * @jest-environment jsdom
  */
 
+/**
+ * External dependencies
+ */
+import wpcomRequest from 'wpcom-proxy-request';
+
 /**
  * Internal dependencies
  */
-import { wpcomRequest as wpcomRequestOriginal } from '../../utils';
 import { createControls } from '../controls';
 
-jest.mock( '../../utils', () => ( {
-	wpcomRequest: jest.fn(),
+jest.mock( 'wpcom-proxy-request', () => ( {
+	__esModule: true,
+	default: jest.fn(),
+	requestAllBlogsAccess: jest.fn( () => Promise.resolve() ),
 } ) );
```

Comment:
A centralized set of generic controls for wpcom-proxy-request and generic fetch could make this testing easier.

Reply:
Follow-up material maybe? :grimacing: